### PR TITLE
Add release note for 20201012

### PR DESCRIPTION
**Pull request by tennix**
### What is the related PR or file link(s)?

Reference link(s):

### CODE REVIEW

File: release-notes.md [hunk 1]
```diff
@@ -7,6 +7,17 @@ summary: Learn about the release notes of TiDB Cloud.
 
 This page lists the release notes of [TiDB Cloud](https://pingcap.com/products/tidbcloud).
 
+## October 12, 2020
+
+* Change SQL webshell client from Oracle MySQL client to usql
+* Upgrade default TiDB version to 4.0.7
```
Comment (lilin90) [inline release-notes.md:13]:
```suggestion
* Upgrade the default TiDB version to 4.0.7
```

File: release-notes.md [hunk 2]
```diff
@@ -7,6 +7,17 @@ summary: Learn about the release notes of TiDB Cloud.
...
+* Tune TiDB default configuration for t1.standard and h1.standard
```
Comment (lilin90) [inline release-notes.md:14]:
```suggestion
* Tune default TiDB configuration for t1.standard and h1.standard
```

File: release-notes.md [hunk 3]
```diff
@@ -7,6 +7,17 @@ summary: Learn about the release notes of TiDB Cloud.
...
+* Change manual backup reserve days from 7 to 30
```
Comment (lilin90) [inline release-notes.md:15]: Please check whether this is what you mean.
```suggestion
* Extend the manual backup retention period from 7 days to 30 days
```

**Review by lilin90 — APPROVED** [review]
LGTM

**Review by sivagao — APPROVED** [review]

**Pull request merged**